### PR TITLE
Upgrade README.md on chromedriver

### PR DESCRIPTION
**Pull request by david-mears-dfe**
This is because I had the following issue:

> brew update --cask chromedriver

> Error: This command updates brew itself, and does not take formula names.
> Use brew upgrade --cask chromedriver instead.

### CODE REVIEW

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-3024.london.cloudapps.digital>

File: README.md [hunk 1]
```diff
@@ -69,6 +69,8 @@ which chromedriver
 xattr -d com.apple.quarantine /path/to/chromedriver
 ```
 
+If this has no effect, you may need to try another path, for example: `/usr/local/bin/chromedriver`
```
Comment (csutter) [inline README.md:72]: 😕  `/path/to` is just a placeholder. Not a big Homebrew fan in general, but I know I'm in a total minority amongst Mac users and we assume it elsewhere in our READMEs, so let's just replace the placeholder with `/usr/local/bin`?

Comment (david-mears-dfe) [reply README.md:72]: I realise that /path/to is a placeholder, but when I replaced it with the output of `which chromedriver`, there was no effect.

Comment (david-mears-dfe) [reply README.md:72]:
```bash
$ which chromedriver
/Users/daivdmears/.local/bin/chromedriver
```

Comment (csutter) [reply README.md:72]: Ooh interesting – how did you end up with two Chromedrivers! And even more interestingly, how is this one before the Homebrew one in the `PATH`, yet the Homebrew one is the one used by your tests 🧐

Comment (david-mears-dfe) [reply README.md:72]: Maybe by being silly. I'll undo this part of the changes to the README anyway.

**Review by csutter — APPROVED** [review]

**Pull request merged**